create_framework/dependency_injection.rst

```diff
@@ -10,7 +10,6 @@ to it::
     namespace Simplex;
 
     use Symfony\Component\EventDispatcher\EventDispatcher;
-    use Symfony\Component\HttpFoundation;
```

Comment:
We have another use like this for HttpFoundation further below. Maybe you can clean that up too?

Reply:
If you refer to L107, it's used in code `$container->register('request_stack', HttpFoundation\RequestStack::class);`